.github/workflows/nms-workflow.yml

```diff
@@ -0,0 +1,254 @@
+            sudo apt-get install -y google-chrome-stable libxss1 --no-install-recommends
+            sudo rm -rf /var/lib/apt/lists/*
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install # will run `yarn install` command
+      - name: run e2e_test_setup.sh
+        run: |
+            source $NVM_DIR/nvm.sh
+            nvm install stable
+            cd ${MAGMA_ROOT}/nms/packages/magmalte
+            ./e2e_test_setup.sh
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
+        with:
+          check_name: ${{ github.event.workflow.name }}
+          commit: ${{ github.event.workflow_run.head_sha }}
+          files: "/tmp/nms_artifacts/*"
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "NMS nms-e2e-test failed"
+          SLACK_MESSAGE: "${{ steps.commit.outputs.title}}"
+          SLACK_USERNAME: "NMS workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+          SLACK_FOOTER: ' '
+  nms-build:
+    needs:
+      - path_filter
+      - nms-flow-test
+      - nms-eslint
+      - nms-yarn-test
+      - nms-e2e-test
+    if: ${{ needs.path_filter.outputs.should_skip == 'false' }}
+    name: nms-build job
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      NMS_ROOT: "${{ github.workspace }}/nms/packages/magmalte"
+      DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
+      DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
+      DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run docker compose
+        id: nms-docker-compose
+        # yamllint disable rule:line-length
+        run: |
+          cd ${MAGMA_ROOT}/nms/packages/magmalte
+          docker-compose build magmalte
+      - name: Tag and push to Docker Registry
+        if: github.ref == 'refs/heads/master' && github.event_name != 'pull_request'
+        # yamllint disable rule:line-length
+        run: |
+            ./ci-scripts/tag-push-docker.sh --images 'magmalte' --tag "${GITHUB_SHA:0:8}" --tag-latest false --project magmalte
+      - name: Tag and push to Jfrog Registry
+        if: github.ref == 'refs/heads/master'
+        env:
+          DOCKER_REGISTRY: "${{ secrets.JFROG_DOCKER_ORC8R_REGISTRY }}"
+          DOCKER_USERNAME: "${{ secrets.JFROG_USERNAME }}"
+          DOCKER_PASSWORD: "${{ secrets.JFROG_PASSWORD }}"
+        run: |
+            ./ci-scripts/tag-push-docker.sh --images 'magmalte' --tag "${GITHUB_SHA:0:8}" --tag-latest true --project magmalte
+      - name: Extract commit title
+        id: commit
+        if: github.ref == 'refs/heads/master'
```

Comment:
On all extract commit title step and slack notification:
Could you replace `github.ref == 'refs/heads/master'` with `github.event_name =='push'` .

Reply:
sure!